invenio_access/__init__.py

```diff
@@ -142,6 +142,45 @@
 >>> admin_identity.can(permission_index)
 True
 
+How to discover existing actions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Modules that intend to implement a given set of actions can register them in
+entry points in the corresponding `setup.py`, e.g.
+
+.. code-block:: python
+
+    entry_points={
+        'invenio_access.actions': [
+            'records_read_all'
+            ' = invenio_records.permissions:records_read_all',
+            'records_create_all'
+            ' = invenio_records.permissions:records_create_all',
+            'records_update_all'
+            ' = invenio_records.permissions:records_update_all',
+            'records_delete_all'
+            ' = invenio_records.permissions:records_delete_all',
+        ],
+    }
+
+In order to discover which actions are available in a given installation one
+can retrieve them via:
+
+>>> sorted(app.extensions['invenio-access'].actions.keys())
+['admin-access', 'superuser-access']
+
+.. note::
+    One can use CLI to discover registered actions. Here's an example:
+
+.. code-block:: console
```

Comment:
![image](https://cloud.githubusercontent.com/assets/189798/17486717/263698b0-5d92-11e6-995c-c3dce4e99d4d.png)

Shall the example be included inside the note box? Or maybe don't need to Note box. WDYT?